### DISCUSSION TITLE
AddLiquidFilter not working

### DISCUSSION THREAD

**Discussion by datdv260497** [OrchardCMS/OrchardCore#10496] [Q&A]
Hi  !. I create a liquid filter, but I get error 
 Shape type 'find_term' not found
   at OrchardCore.DisplayManagement.Implementation.DefaultHtmlDisplay.ExecuteAsync(DisplayContext context)

![image](https://user-images.githubusercontent.com/49801948/137426755-8e739be1-9c01-448f-8076-aeeb6f567452.png)
![image](https://user-images.githubusercontent.com/49801948/137426780-878cbe8b-56a6-4388-b3b6-b76203e52da8.png)
Is there a problem with my configuration?
Thank !

**Comment by ns8482e** [comment]
It seems the error is not related to registration of liquid filter - may be an error within your filter code itself.
